contrib/init/datum_gateway.service

```diff
@@ -0,0 +1,89 @@
+[Unit]
+Description=DATUM Gateway
+Documentation=https://github.com/OCEAN-xyz/datum_gateway
+
+#
+# Expected file locations when using hardening options:
+# binary: /usr/bin/datum/
+# configuration: /etc/datum/
+# logs: /var/lib/datum_gateway/
+#
+
+# https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/
+Wants=network-online.target
+After=network-online.target
+# alternatively, wait until bitcoind is up
+#After=bitcoind.service
+
+[Service]
+
+ExecStart=/usr/bin/datum/datum_gateway --config=/etc/datum/datum_gateway_config.json
```

Comment:
Why would the binary be in a subdirectory?

Reply:
Yeah, you're right, no good reason (inherited from an earlier config that's no longer relevant on my system). I'll fix that and update.